src/main/java/de/retest/web/PeerConverter.java

```diff
@@ -81,7 +90,7 @@ private WebElementPeer convertToPeer( final String path, final WebData webData )
 	}
 
 	protected boolean isRoot( final String parentPath ) {
-		return parentPath == null;
+		return parentPath == null || parentPath.equals( rootParentPath );
```

Comment:
Do we want this as root, since a [screen and title seems awkward](https://github.com/retest/recheck-web/pull/338/files#diff-2c55711e8071f1b366e12fc1bf319442R4) on a single WebElement. Don't we just want to wrap this into a `RootElement`, as we have a driver.

Reply:
I agree. But I think this is a problem to be addressed in the data model (i.e. in the `RootElement`) rather than here. We should not make another bad decision because of an earlier bad decision...